aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/handlers/widget/WidgetHandlerUtils.java

```diff
@@ -44,40 +42,40 @@
  * collections, such as {@link com.exadel.aem.toolkit.api.annotations.widgets.FieldSet}
  * or {@link com.exadel.aem.toolkit.api.annotations.widgets.MultiField}
  */
-interface WidgetSetHandler extends Handler, BiConsumer<Element, Field> {
+interface WidgetHandlerUtils {
```

Comment:
Please consider dismissing this interface as described above

Reply:
Interface was deleted